cardano-node/src/Cardano/Node/Platform/Posix.hs

```diff
@@ -0,0 +1,3 @@
+module Cardano.Node.Platform.Posix
```

Comment:
Maybe make compilation of this module conditional in the `.cabal` file? Same for `Window.hs`. Because as soon as you add platform-specific code/dependencies, it will cause compilation issues on the other platform.